babel/src/board.js

```diff
@@ -283,20 +280,29 @@ export class Board {
         let hasWideStyleMatch = _.some(_.map(flatChunks, hasMatchInSingleRow));
         return hasWideStyleMatch || _.some(_.map(chunks), hasMatchInPairOfRows);
     }
+    
+    hasMatchEvent() {
+        if (findMatches(this.orbs)[0]) { return true };
```

Comment:
Right but when it gets called in an if statement wherever, it will then be considered truthy. I get it.